package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ssb-validate2-rsjs-node",
-  "version": "0.5.1",
+  "version": "0.6.0",
```

Comment:
@mycognosist Actually, I just realized, I think this should not be bumped yet. It should be bumped only its own commit message that starts with `"release"`, e.g. `git commit -m'release 0.6.0'` which will trigger CI to do the version release for us.

Reply:
Ah yes, good call. I made a mess of the revert but it's resolved now.